porcupine/plugins/highlight/tree-sitter-token-mappings/python.yml

```diff
@@ -6,49 +6,109 @@ dont_recurse_inside:
 
 token_mapping:
   comment: Token.Comment
+  ERROR: Token.Error
 
   # TODO: which of these are needed?
   integer: Token.Literal.Number.Integer
   float: Token.Literal.Number.Float
   string: Token.Literal.String
 
-  and: Token.Keyword
+  'true': Token.Keyword.Constant
+  'false': Token.Keyword.Constant
+  none: Token.Keyword.Constant
+  
+  ellipsis: Token.Name.Builtin.Pseudo
+  def_name: Token.Name.Function
+  class_name: Token.Name.Class
+
+  # Keywords
   as: Token.Keyword
   assert: Token.Keyword
   async: Token.Keyword
   await: Token.Keyword
   break: Token.Keyword
+  case: Token.Keyword
   class: Token.Keyword
   continue: Token.Keyword
   def: Token.Keyword
   del: Token.Keyword
   elif: Token.Keyword
   else: Token.Keyword
   except: Token.Keyword
-  'false': Token.Keyword
   finally: Token.Keyword
   for: Token.Keyword
   from: Token.Keyword
   global: Token.Keyword
   if: Token.Keyword
   import: Token.Keyword
-  in: Token.Keyword
-  is: Token.Keyword
   lambda: Token.Keyword
-  none: Token.Keyword
+  match: Token.Keyword
   nonlocal: Token.Keyword
-  not: Token.Keyword
-  or: Token.Keyword
   pass: Token.Keyword
   raise: Token.Keyword
   return: Token.Keyword
-  'true': Token.Keyword
   try: Token.Keyword
   while: Token.Keyword
   with: Token.Keyword
   yield: Token.Keyword
 
+  # Operators
+  '!=': Token.Operator
+  '==': Token.Operator
+  '<': Token.Operator
+  '>': Token.Operator
+  '<=': Token.Operator
+  '>=': Token.Operator
+  '+=': Token.Operator
+  '-=': Token.Operator
+  '*=': Token.Operator
+  '/=': Token.Operator
+  '@=': Token.Operator
+  '//=': Token.Operator
+  '%=': Token.Operator
+  '**=': Token.Operator
+  '<<=': Token.Operator
+  '>>=': Token.Operator
+  '&=': Token.Operator
+  '^=': Token.Operator
+  '|=': Token.Operator
+  '+': Token.Operator
+  '-': Token.Operator
+  '*': Token.Operator
+  '/': Token.Operator
+  '@': Token.Operator
+  '//': Token.Operator
+  '%': Token.Operator
+  '**': Token.Operator
+  '<<': Token.Operator
+  '>>': Token.Operator
+  '&': Token.Operator
+  '^': Token.Operator
+  '|': Token.Operator
+  '~': Token.Operator
+  '=': Token.Operator
+  ':=': Token.Operator
+  and: Token.Operator.Word
+  in: Token.Operator.Word
+  is: Token.Operator.Word
+  not: Token.Operator.Word
+  or: Token.Operator.Word
```

Comment:
I don't like `Token.Operator.Word`. Not all themes define a color for it, and that makes the themes quite unusable on pygments: try `assert x is not None` with the `fruity` theme. I also have never understood why some keywords should be highlighted separately from others: I think of them as keywords, not as operators, even if they technically are operators.